RecoLocalMuon/GEMSegment/python/gemSegments_cfi.py

```diff
@@ -1,10 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
+# defaults for GE0SegAlgoRU
+RU_GE0 = cms.PSet(
+    allowWideSegments = cms.bool(True),
+    doCollisions = cms.bool(True),
+    maxChi2Additional = cms.double(100.0),
+    maxChi2Prune = cms.double(50),
+    maxChi2GoodSeg = cms.double(50),
+    maxPhiSeeds = cms.double(0.001096605744), #Assuming 384 strips
+    maxPhiAdditional = cms.double(0.001096605744), #Assuming 384 strips
+    maxETASeeds = cms.double(0.1), #Assuming 8 eta partitions
+    maxTOFDiff = cms.double(25),
+    requireCentralBX = cms.bool(True), #require that a majority of hits come from central BX
+    minNumberOfHits = cms.uint32(4),
+    maxNumberOfHits = cms.uint32(300),
+    maxNumberOfHitsPerLayer = cms.uint32(100),
+)
+
```

Comment:
Why did you add `RU_GE0` in `gemSegments_cfi.py`? This file should contain only the default values of `gemSegments`

Reply:
Because this is the default for the RU algorithm, which is the default reconstruction for ME0. For GE0, we will have a student check the various segment algorithms and possibly add the option to change algorithm by station, so I wanted to have this here to make it easy for them to switch algorithms.